pipettes/firmware/motor_timer_hardware.c

```diff
@@ -15,6 +18,47 @@ static gear_motor_interrupt_callback gear_callback = NULL;
 // /1 AHB
 // /2 APB1
 // /425 prescaler * 1 count = 200kHz
+
+void TIM2_Encoder_Init(void) {
```

Comment:
Can we check the 96 chan schematic and make sure we're also supporting whatever we need to for the encoder?

Reply:
checked on this- pins are the same for the 96 channel